BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -69,30 +69,44 @@ private struct SearchableVaultListView: View {
 
     /// A view that displays the empty vault interface.
     @ViewBuilder private var emptyVault: some View {
-        GeometryReader { reader in
-            ScrollView {
```

Comment:
⚠️ I think we still want the scroll view so it scrolls on smaller devices or with large fonts.

<img width="559" alt="Screenshot 2024-10-08 at 9 35 12 AM" src="https://github.com/user-attachments/assets/b741558c-a66e-47e0-8efe-5c9850421431">